src/libraries/System.Private.CoreLib/src/System/IO/File.cs

```diff
@@ -689,6 +689,62 @@ public static void WriteAllBytes(string path, byte[] bytes)
             RandomAccess.WriteAtOffset(sfh, bytes, 0);
         }
 
+        /// <summary>
+        /// Appends the specified byte array to the end of the file at the given path.
+        /// If the file doesn't exist, this method creates a new file.
+        /// </summary>
+        /// <param name="path">The file to append to.</param>
+        /// <param name="bytes">The bytes to append to the file.</param>
+        /// <exception cref="System.ArgumentException">
+        /// Thrown when <paramref name="path"/> is a zero-length string, contains only white space,
+        /// or contains invalid characters as defined by <see cref="System.IO.Path.GetInvalidPathChars"/>.
+        /// </exception>
```

Comment:
Don't use "Throw when".
```suggestion
        /// <exception cref="System.ArgumentException">
        /// <paramref name="path"/> is a zero-length string, contains only white space, or contains one more invalid characters defined by the <see cref="System.IO.Path.GetInvalidPathChars"/> method.
        /// </exception>
```